volatility/framework/plugins/windows/cmdline.py

```diff
@@ -30,25 +30,39 @@ def get_requirements(cls) -> List[interfaces.configuration.RequirementInterface]
                                          optional = True)
         ]
 
+    @classmethod
```

Comment:
Also, I realized I forgot one thing.  Now that this is a classmethod and so exposes an API, the class should define a:
```
_version = (1, 0, 0)
```
So that people can add a plugin requirement against it...